utils/generate_types/overrides-test.d.ts

```diff
@@ -77,6 +77,11 @@ export type WebServerConfig = {
    * The port that your http server is expected to appear on. It does wait until it accepts connections.
    */
   port: number,
+  /**
+   * The url on your http server that is expected to return a 2xx status code when the server is ready to accept connections.
+   * If the url is not specified, only the port is checked.
+   */
+  url?: string,
```

Comment:
- Let's make `port` optional.
- Please update comment for both port and url to mention that exactly one of them is required, and align the implementation.
- Please update reuseExistingServer comment to equally mention port and url.
- Please also update webServer section in `docs/src/test-api/class-testconfig.md`.